crates/polymesh-api-tester/src/lib.rs

```diff
@@ -1,7 +1,5 @@
 #[cfg(feature = "polymesh_v7")]
-pub use polymesh_api::polymesh::types::polymesh_primitives::{
-  asset::AssetId,
-};
+pub use polymesh_api::polymesh::types::polymesh_primitives::asset::AssetId;
 pub use polymesh_api::{
   client::{AccountId, IdentityId, Signer},
```

Comment:
The `AssetId` type should be coming from `polymesh_api::client` like `IdentityId`.  Also need to update the codegen type mapper like what is done for `IdentityId`: https://github.com/PolymeshAssociation/polymesh-api/blob/fb88551463f2d8579b7a459107387c6777e2e8c9/crates/polymesh-api-codegen/src/generate.rs#L205